ScienceDirect.js

```diff
@@ -355,11 +355,25 @@ function scrape(doc) {
 		return;
 	}
 	
+	
+	// On newer pages, there is an GET formular which is only there if
+	// the user click on the export button, but we know how the url
+	// in the end will be built.
+	form = ZU.xpath(doc, '//div[@class="ExportCitation"]//a[contains(@class, "ExportCitattionButton")]');
+	if (form) {
```

Comment:
Okay, yes. That is also the reason that my typo above had any influence on the result. I will change `form = ...[0];` as in the first block here.